FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth/DemoTasks/MutualAuthMQTTExample.c

```diff
@@ -497,13 +528,25 @@ static TlsTransportStatus_t prvConnectToServerWithBackoffRetries( NetworkCredent
     /* Set the credentials for establishing a TLS connection. */
     pxNetworkCredentials->pRootCa = ( const unsigned char * ) democonfigROOT_CA_PEM;
     pxNetworkCredentials->rootCaSize = sizeof( democonfigROOT_CA_PEM );
+    /* SNI needs to be disabled for a local Mosquitto server. */
```

Comment:
```suggestion
    /* SNI needs to be disabled for a non-AWS IoT Core MQTT broker. */
```

Reply:
Actually, this logic should go in the `#else` block for `#ifdef democonfigUSE_AWS_IOT_CORE_BROKER` to make it easier for the reader to understand that this setting is only used for non-AWS IoT connections